.github/workflows/publish-site.yml

```diff
@@ -0,0 +1,41 @@
+name: Publish Armeria site
+
+on:
+  push:
+    tags:
+      - armeria-*
+
+env:
+  LC_ALL: "en_US.UTF-8"
```

Comment:
Not a big deal. Just curious. `"` and `'` are mixed to represent strings such `"en_US.UTF-8"` and `'temurin'`. How about keeping consistency?